tests/modules/services/redshift-gammastep/gammastep-basic-configuration-file-expected.conf

```diff
@@ -1,11 +1,13 @@
 [general]
 adjustment-method=randr
+dawn-time=6:00-7:45
+dusk-time=18:35-20:15
 gamma=0.800000
-latitude=0.0
 location-provider=manual
-longitude=0.000000
 temp-day=5500
 temp-night=3700
 
+[manual]
```

Comment:
This empty section doesn't seem to be a problem with `redshift` or `gammastep`. They happily accept it.